Demo/Common/ViewController.swift

```diff
@@ -705,6 +707,42 @@ extension ViewController {
         )
     }
     
+    // MARK: Jira
+    func doOAuthJira(_ serviceParameters: [String:String]){
+        assert(Bundle.main.path(forResource: "private", ofType: "pem") != nil, "Jira authentication uses RSA-SHA1 which requires a stighlty different setup - see README.md for details.")
+        let oauthswift = OAuth1Swift.initWithRSA(
```

Comment:
could add something like that 
```swift
let jiraURL = serviceParameters["url"] ?? "https://yourinstance.atlassian.net"
```
and then use it in the following code, allowing to use in config file a custom url